workos/utils/_base_http_client.py

```diff
@@ -28,12 +21,17 @@
 DEFAULT_REQUEST_TIMEOUT = 25
 
 
+ParamsType = Optional[Mapping[str, Any]]
+HeadersType = Optional[Dict[str, str]]
+ResponseJson = Dict[Any, Any]
```

Comment:
Why are some `Mapping` and some `Dict`?

Reply:
Good question. We don't set items after assignment on params currently (Mapping), but we do for headers (Dict). ResponseJson should be a Mapping.